src/components/organisms/UiPhoneNumber/_internal/UiPhoneNumberPrefix.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <UiDropdown
```

Comment:
oki doki; are your check [this stories ](https://component.infermedica.com/?path=/story/molecules-dropdown--as-select)?
You can set a lot on UiDropdown
```
<UiDropdown
      v-model="modelValue"
      :text="modelValue"
      :name="name"
      :close-on-click-outside="closeOnClickOutside"
      :toggle-element="toggleElement"
      :enable-keyboard-navigation="enableKeyboardNavigation"
      :items="items"
      :button-toggle-attrs="buttonToggleAttrs"
      :popover-attrs="popoverAttrs"
      @update:modelValue="onUpdateModelValue"
      @open="onOpen"
      @close="onClose"
    />
```